src/server/page.tsx

```diff
@@ -175,13 +175,16 @@ function page(
     const html = `
         <html lang="en">
             <head>
-                ${renderToString(<Head webTitle={content.webTitle} />)}
+                ${renderToString(<Head webTitle={renderingRequest.content.webTitle} />)}
                 <meta
                     http-equiv="Content-Security-Policy"
                     content="${cspString}"
                 />
                 <style>${styles}</style>
                 <style data-emotion-css="${ids.join(' ')}">${css}</style>
+                <script id="targetingParams" type="application/json">
```

Comment:
Should I put that in the body?

Reply:
Semantically I would say the `head` is the best place, as this is metadata about the document. Performance-wise I'm not aware of any major downsides; the browser shouldn't need to do much with this since it's treated as a [data block](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type).

Reply:
Thanks! 👍